packages/nuxi/src/cli.ts

```diff
@@ -74,7 +74,11 @@ export function main () {
       if (result === 'error') {
         process.exit(1)
       } else if (result !== 'wait') {
-        process.exit(0)
+        if (process.exitCode && process.exitCode >= 1) {
+          process.exit(process.exitCode)
+        } else {
+          process.exit(0)
+        }
```

Comment:
Is there any reason not to always all `.exit` with the exitCode?

```suggestion
        process.exit(process.exitCode)
```

Reply:
I thing setting the exit code can be emitted all together because the default is 0 and then other processes can overwritten by process.exitCode. That’s what vitest tries todo.

Reply:
From the manual: Specifying a code to [process.exit(code)](https://nodejs.org/api/process.html#processexitcode) will override any previous setting of process.exitCode.